src/main/kotlin/org/jitsi/nlj/Transceiver.kt

```diff
@@ -270,9 +266,8 @@ class Transceiver(
     fun setSrtpInformation(chosenSrtpProtectionProfile: Int, tlsRole: TlsRole, keyingMaterial: ByteArray) {
         val srtpProfileInfo =
             SrtpUtil.getSrtpProfileInformationFromSrtpProtectionProfile(chosenSrtpProtectionProfile)
-        logger.cinfo { "Transceiver $id creating transformers with:\n" +
+        logger.cdebug { "Transceiver $id creating transformers with:\n" +
                 "profile info:\n$srtpProfileInfo\n" +
-                "keyingMaterial:\n${keyingMaterial.toHex()}\n" +
```

Comment:
Can we just comment this out maybe? Or use a different log level? It's needed to create a replay able pcap for the harnesses